openvino_tensorflow/ie_basic_engine.cc

```diff
@@ -30,11 +36,23 @@ void IE_Basic_Engine::infer(
     std::vector<std::string>& output_names,
     std::vector<std::shared_ptr<IETensor>>& hoisted_params,
     std::vector<std::string>& param_names) {
+  int64_t start_ns = GetCurrentTimeNanos();
   load_network();
+  if (std::getenv("OPENVINO_TF_ENABLE_INFERENCE_TIMING")) {
```

Comment:
Same as above about the environment variables. Otherwise, we will be reading this for each iteration of each cluster.